scripts/deploy/govcms-prepare

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS prepare for deploy.
+#
+# This performs some perfunctory commands to ensure that the
+# sites are in a deployable position.
+#
+
+sed -i "s/%%PROJECT_NAME%%/\${LAGOON_PROJECT}/g" /app/drush/sites/govcms.site.yml
```

Comment:
If all sites are using the latest image then this script might now be redundant. The alias works differently now.
https://github.com/govCMS/govcms8lagoon/blob/develop/.docker/images/govcms8/govcms.site.yml

So making a whole script for this one deprecated line is too much IMO

Reply:
Yeah - agree I didn't really audit them as I moved them over. Probably should have I was going for parity first and then hoping we could revist and update!

Reply:
Yeah agreed, just a deprecation notice on the while script means we capture the knowledge at least.